### DISCUSSION TITLE
Is `bootstrap-vue-3` the way forward for vue3 / Bootstrap 5 (the future of `bootstrap-vue`)?

### DISCUSSION THREAD

**Discussion by ceisele-r** [cdmoro/bootstrap-vue-3#874] [Q&A] [answered]
Hi,

Following the activity over the past months/years, I'm currently a little bit confused about the status of the different bootstrap-vue repositories and would like to clean this up.

Will this repo (`bootstrap-vue/bootstrap-vue-next` / npm package `bootstrap-vue-3`) be the way forward in terms of vue3 / bootstrap 5 support?
Or will this just be an interim repo and `bootstrap-vue/bootstrap-vue` (npm package `bootstrap-vue`) will at some point in time release a new major version with full vue3 / bootstrap 5 support?

_Or formulating the question the other way:_
If I now start a new project / a migration to vue 3 (_where I don't care if the used library is still in alpha_), would it be better to start with `bootstrap-vue` (using the compatibility mode) or `bootstrap-vue-3`?

Which will be the way forward? Or will both continue to co-exist with full support for vue3?

I would appreciate any insights, thanks!

**Comment by VividLemon** [comment, marked as answer]
The original boootstrap-vue package will never support Bootstrap v5. It does now support Vue 3, with few Vue 3 specific issues.

Bootstrap-vue-next is not a good option for production apps. There is a small handful of components that are broken, some which will see changes, and some that have breaking changes from Bootstrap-vue that are not documented.

**Reply by ceisele-r** [reply]
Thank you for the fast reply. Is it fair from your comment to assume that `bootstrap-vue-3` would be better to use when starting a new project / migration?
I don't really care about production readiness at the moment as the new project/migration will anyway take some time. So probably `bootstrap-vue-3` would be mature until the project would eventually become production-ready.

Is it fair to assume `bootstrap-vue-3` would be the better choice in this case?

**Reply by VividLemon** [reply]
> Is it fair to assume bootstrap-vue-3 would be the better choice in this case?

I'd say yes.

**Reply by ceisele-r** [reply]
Thanks!

**Reply by marianocodes** [reply]
Hey @VividLemon, is there a timeline or estimate for when `bootstrap-vue-3` be ready?

**Reply by iruizeguilaz** [reply]
Hi @VividLemon , after a year, bootstrap-vue-3 would be the better choice yet? or we are close to chage this answer into ootstrap-vue-next?
